src/kava.js

```diff
@@ -73,6 +75,14 @@ class Kava extends BasePlugin {
       bufferTimeSum: 0.0,
       playTimeSum: 0.0
     });
+
+    // check the Resource Timing API is supported in the browser and we have a uiConfId
+    if (performance && this.config && this.config.uiConfId) {
+      let entries = performance.getEntriesByType('resource').filter(entry => entry.name.match('embedPlaykitJs.*' + this.config.uiConfId));
```

Comment:
you can use `find` instead of `filter` here as you are expecting only one entry in the list.
Find should exit quicker than filter.
Note that `find` returns a result of the type you filter for, and not an array of the results like `filter` does, so you don't need the `entries[entries.length - 1]` just `entry`
```javascript
      let entry = performance.getEntriesByType('resource').find(entry => entry.name.match('embedPlaykitJs.*' + this.config.uiConfId));
      if (entry) {
        this._model.updateModel({playerJSLoadTime: entry.duration});
      }
```